Specs/Scene/ShadowMapSpec.js

```diff
@@ -732,23 +743,42 @@ describe(
       renderAndCall(function (rgba) {
         unshadowedColor = rgba;
         expect(rgba).not.toEqual(backgroundColor);
-      });
+      }, startTime);
 
       // Render with shadows
+      let shadowedColor;
       scene.shadowMap.enabled = true;
       renderAndCall(function (rgba) {
+        shadowedColor = rgba;
         expect(rgba).not.toEqual(backgroundColor);
         expect(rgba).not.toEqual(unshadowedColor);
+
+        // The floor is red, and when shadowed, the red
+        // component should be darker
+        const shadowedRed = rgba[0];
+        const unshadowedRed = unshadowedColor[0];
+        expect(shadowedRed).toBeLessThan(unshadowedRed);
       }, startTime);
 
       // Change the time so that the shadows are no longer pointing straight down
-      renderAndExpect(unshadowedColor, endTime);
+      renderAndCall(function (rgba) {
+        expect(rgba).not.toEqual(backgroundColor);
+        expect(rgba).not.toEqual(shadowedColor);
+        expect(rgba).not.toEqual(unshadowedColor);
+
+        // After changing the sunlight direction, the floor will appear
+        // a bit darker, but not as dark as when it was shadowed by the box
+        const red = rgba[0];
+        const unshadowedRed = unshadowedColor[0];
+        const shadowedRed = shadowedColor[0];
+        expect(shadowedRed).toBeLessThan(red);
```

Comment:
this may seem nitpicky, but I think this reads better:

```suggestion
        expect(red).toBeGreaterThan(shadowedRed);
```